functional/lcow_test.go

```diff
@@ -22,7 +23,91 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
-func TestV2XenonLCOW(t *testing.T) {
+// TestLCOWUVMNoSCSISingleVPMem starts an LCOW utility VM without a SCSI controller and
+// only a single VPMem device. Uses initRD
+func TestLCOWUVMNoSCSISingleVPMem(t *testing.T) {
+	opts := &uvm.UVMOptions{
+		OperatingSystem:  "linux",
+		ID:               "uvm",
+		VPMemDeviceCount: 1,
+		NoSCSI:           true,
+	}
+	lcowUVM, err := uvm.Create(opts)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := lcowUVM.Start(); err != nil {
+		t.Fatal(err)
+	}
+	defer lcowUVM.Terminate()
+	out, err := exec.Command(`hcsdiag`, `exec`, `-uvm`, lcowUVM.ID(), `dmesg`).Output()
```

Comment:
I dont know this infra very well yet. Why are we using hcsdiag and not use the HCS CreateProcess call ourselves?

Reply:
It's a temporary hack. I'll add a TODO here.